disdrodb/tests/test_L0_readers.py

```diff
@@ -0,0 +1,293 @@
+
+    list_campaigns_paths = glob.glob(os.path.join(path_reader, "*", "*"))
+    dict_available_testing_items = {}
+
+    # Iterate through campaigns
+    for campaign_path in list_campaigns_paths:
+        zip_files = [
+            f
+            for f in os.listdir(campaign_path)
+            if f in [f"{i}.zip" for i in list_zip_files_names]
+        ]
+
+        # Check if the folder contains  "raw.zip" AND "processed.zip" files
+        if sorted(zip_files) == sorted([f"{i}.zip" for i in list_zip_files_names]):
+            campaign_name = os.path.basename(campaign_path)
+            data_source_name = os.path.basename(os.path.dirname(campaign_path))
+
+            # Unzip files
+            for item in list_zip_files_names:
+                source = os.path.join(campaign_path, f"{item}.zip")
+                dest = os.path.join(
+                    LOCAL_FOLDER,
+                    "testing_files",
+                    "DISDRODB",
+                    item.title(),
+                    data_source_name,
+                    campaign_name,
+                )
+                with zipfile.ZipFile(source, "r") as zipObj:
+                    zipObj.extractall(dest)
+
+            # Populate the dictionnary of available testing items
+            if not data_source_name in dict_available_testing_items.keys():
+                dict_available_testing_items[data_source_name] = []
+            dict_available_testing_items[data_source_name].append(campaign_name)
+
+    return dict_available_testing_items
+
+
+def run_reader_on_test_data(data_source: str, campaign_name: str) -> None:
+    """Run reader on test data
+
+    Parameters
+    ----------
+    data_source : str
+        Data source
+    campaign_name : str
+        Name of the campaign
+    """
+
+    raw_dir = os.path.join(
+        LOCAL_FOLDER, "testing_files", "DISDRODB", "Raw", data_source, campaign_name
+    )
+    processed_dir = os.path.join(
+        LOCAL_FOLDER,
+        "test_results",
+        "DISDRODB",
+        "Processed",
+        data_source,
+        campaign_name,
+    )
+    l0a_processing = True
+    l0b_processing = True
+    keep_l0a = True
+    force = True
+    verbose = True
+    debugging_mode = True
+    lazy = False
+    single_netcdf = True
+
+    run_reader(
+        data_source,
+        campaign_name,
+        raw_dir,
+        processed_dir,
+        l0a_processing,
+        l0b_processing,
+        keep_l0a,
+        force,
+        verbose,
+        debugging_mode,
+        lazy,
+        single_netcdf,
+    )
+
+
+def are_netcdf_identital(dataset_1: Dataset, dataset_2: Dataset) -> bool:
+    """Check if two NetCDF are identical
+
+    Parameters
+    ----------
+    dataset_1 : Dataset
+        First NetCDF
+    dataset_2 : Dataset
+        Second NetCDF
+
+    Returns
+    -------
+    bool
+        True if identical,
+        False if not indentical
+    """
+
+    is_identical = True
+
+    # Check dimension keys

```

Comment:
Is it not worth using one of the following functions? 
https://docs.xarray.dev/en/stable/generated/xarray.testing.assert_allclose.html 
https://docs.xarray.dev/en/stable/generated/xarray.testing.assert_equal.html
https://xarray.pydata.org/en/v2022.09.0/generated/xarray.testing.assert_identical.html